config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
Yep, I get it. Just trying to fix the URL errors in the RSS validation - https://validator.w3.org/feed/check.cgi?url=http%3A%2F%2Fipfs.io%2Fblog%2Findex.xml in response to https://github.com/ipfs/blog/issues/22#issuecomment-323340950

I'm open to suggestions.